src/kolmogorov_solve.jl

```diff
@@ -39,12 +40,23 @@ function DiffEqBase.solve(
     chain  = alg.chain
     opt    = alg.opt
     sdealg = alg.sdealg
+    ensemblealg = alg.ensemblealg
     ps     = Flux.params(chain)
     xi     = rand(xs ,d ,N[1])
     #Finding Solution to the SDE having initial condition xi. Y = Phi(S(X , T))
     sdeproblem = SDEProblem(μ,sigma,xi,tspan,noise_rate_prototype = noise_rate_prototype)
-    sol = solve(sdeproblem, sdealg ,dt=0.01 , save_everystep=false , kwargs...)
-    x_sde = sol[end]
+    function prob_func(prob,i,repeat)
+      SDEProblem(prob.f , prob.g , xi[: , i] , prob.tspan ,noise_rate_prototype = prob.noise_rate_prototype)
+    end
+    output_func(sol,i) = (sol[end],false)
+    ensembleprob = EnsembleProblem(sdeproblem , prob_func = prob_func , output_func = output_func)
+    sim = solve(ensembleprob, sdealg, ensemblealg , dt=0.01,trajectories=N[1],adaptive=false)
```

Comment:
you'll still want to do `save_everystep=false,save_start=false` to reduce the memory usage.